src/components/Contacts/ContactDetails/ContactTasksTab/ContactTasksTab.test.tsx

```diff
@@ -303,7 +323,7 @@ describe('ContactTasksTab', () => {
 
     // Wait for the fetchMore to be called
     await waitFor(() => {
-      const { operation } = querySpy.mock.calls[2][0];
+      const { operation } = querySpy.mock.calls[4][0];
```

Comment:
Every time I see changes like this updating the call index of spies, I think of how it would be more future-proof to just use `expect().toHaveGraphqlOperation()`. Maybe whenever there are tests like this that need updated, we switch to `toHaveGraphqlOperation` to slowly migrate our tests over to that more robust solution.